src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -580,6 +586,129 @@ export class HaAutomationEditor extends PreventUnsavedMixin(
     }
   }
 
+  public connectedCallback() {
+    super.connectedCallback();
+    window.addEventListener("paste", this._handlePaste);
+  }
+
+  public disconnectedCallback() {
+    window.removeEventListener("paste", this._handlePaste);
+    super.disconnectedCallback();
+  }
+
+  private _handlePaste = async (ev: ClipboardEvent) => {
+    // Ignore events on inputs/textareas
+    const target = ev.composedPath()[0];
```

Comment:
We should instead re-use this logic (extract it to a helper function) https://github.com/jpbede/home-assistant-frontend/blob/4d9aad1f8e940d135dc7216bb58c1c576991b24d/src/state/quick-bar-mixin.ts#L189